crates/autopilot/src/domain/competition/participant.rs

```diff
@@ -1,62 +1,88 @@
-use {
-    super::{Score, Solution},
-    crate::infra,
-    std::sync::Arc,
-};
+use {super::Score, crate::infra, std::sync::Arc};
 
 #[derive(Clone)]
 pub struct Participant<State = Ranked> {
-    solution: Solution,
+    solution: super::Solution,
```

Comment:
You import super::Score, so I think you import super::Solution and just use "Solution" here. 
```suggestion
    solution: Solution,
```